docs/meshcloud.index.md

```diff
@@ -71,3 +71,8 @@ We are interested to make that work as well!
 
 This documentation is open source! Please feel free to hit the `Edit` button any time and help us [improve](https://github.com/meshcloud/meshcloud-docs/blob/master/CONTRIBUTING.md) the documentation. Your feedback is very welcome.
 
+## Getting Updates
+
+We release on a regular basis and provide weekly updates. If you want to stay on top changes in meshStack please have a look at the roadmap on the webside under [www.meshcloud.io](https://www.meshcloud.io/en/product/) or have a look at the release notes above.
+Follow (https://docs.meshcloud.io/blog) which provides atom and rss feeds. Or follow either one of (https://docs.meshcloud.io/blog/feed.xml) and (https://docs.meshcloud.io/blog/atom.xml) directly. 
```

Comment:
I propose the following. I also added the newsletter & made it bullet points so it's easier to process.

```suggestion
We release regularly and provide weekly updates. If you want to stay on top of changes in meshStack you can use the following: 

- Have a look at the roadmap on the website under [www.meshcloud.io](https://www.meshcloud.io/en/product/).
- Subscribe to the [Product Newsletter](https://share.hsforms.com/1AbELCsdRRP6EaCkm1UeATwc0hrp) which is sent out every two weeks summarizing the most important changes.
- Have a look at the [release notes](https://docs.meshcloud.io/blog) or follow either one of [RSS](https://docs.meshcloud.io/blog/feed.xml) or [atom](https://docs.meshcloud.io/blog/atom.xml) directly. 
```